app/models/subject_queue.rb

```diff
@@ -79,6 +79,10 @@ def below_minimum?
   end
 
   def next_subjects(limit=10)
-    set_member_subject_ids[0..limit-1]
+    if user_id
+      set_member_subject_ids[0..limit-1]
```

Comment:
Sure we could do that. I don't think any of it really matters though since we're still retiring once a certain classification count is reached. I think it's probably a good think to directly user attention to few subjects at once as that's likely to produce more discussion on talk.